services/net/clip/ClipAction.cs

```diff
@@ -92,7 +93,9 @@ public ClipAction(IApiService api, IOptions<ClipOptions> options, ILogger<ClipAc
                     {
                         // TODO: Waiting for each clip to complete isn't ideal.  It needs to handle multiple processes.
                         await RunProcessAsync(process, cancellationToken);
-                        await this.ContentReceivedAsync(manager, reference, CreateSourceContent(process, manager.Ingest, schedule, reference));
+
+                        reference = await FindContentReferenceAsync(content.Source, content.Uid);
```

Comment:
Why are you requesting another reference here?  You have the latest reference (line 78).  If it enters the if statement then it updates (line 82, or 97).  Don't make extra requests to the API if you don't need to.

Reply:
Removed, however, depends on how long it will take for RunProcessAsync to run, this could be a potential cause I guess? Thanks